packages/grid/_modules_/grid/components/panel/PanelContent.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { classnames } from '../../utils/classnames';
+
+const useStyles = makeStyles(
+  () => ({
+    root: {
+      display: 'flex',
+      flexDirection: 'column',
+      overflow: 'auto',
+      flex: '1 1',
+      maxHeight: 400,
+    },
+  }),
+  { name: 'MuiDataGridPanelContent' },
+);
+
+export function PanelContent(props: React.PropsWithChildren<React.HTMLAttributes<HTMLDivElement>>) {
+  const classes = useStyles();
+  const { children, className, ...otherProps } = props;
+  return (
+    <div className={classnames(classes.root, className)} {...otherProps}>
+      {children}
+    </div>
+  );
```

Comment:
You can use a shorthand (I have also used `other` for consistency).

```suggestion
  const { className, ...other } = props;
  return <div className={classnames(classes.root, className)} {...other} />;
```

x3 (the other Panel components)